backend/YouTubeV2.Application/YTContext.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+using YouTubeV2.Application.EntityConfiguration;
+using YouTubeV2.Application.Model;
+
+namespace YouTubeV2.Application
+{
+    public class YTContext : IdentityDbContext<
```

Comment:
i believe that we don't need custom role and userrole classes. Having custom User class sounds like a good idea though cause we will add videos etc to the user later on